scIB/metrics/metrics.py

```diff
@@ -325,58 +332,50 @@ def metrics(
 
     if kBET_:
         print('kBET...')
-        try:
-            kbet_results = kBET(
-                adata_int,
-                batch_key=batch_key,
-                label_key=label_key,
-                type_=type_,
-                embed=embed,
-                subsample=kBET_sub,
-                heuristic=True,
-                verbose=verbose
-            )
-            kbet_score = 1 - np.nanmean(kbet_results['kBET'])
-        except NeighborsError:
-            print('Not enough neighbours')
-            kbet_score = 0
+        kbet_score = kBET(
+            adata_int,
+            batch_key=batch_key,
+            label_key=label_key,
+            type_=type_,
+            embed=embed,
+            scaled=True,
+            verbose=verbose
+        )
     else:
         kbet_score = np.nan
 
-    # if lisi_:
-    #    print('LISI score...')
-    #    ilisi_score, clisi_score = lisi(adata_int, batch_key=batch_key, label_key=label_key,
-    #                                    type_ = type_, verbose=verbose)
-    # else:
-    #    ilisi_score = np.nan
-    #    clisi_score = np.nan
-    # results['iLISI'] = ilisi_score
-    # results['cLISI'] = clisi_score
-
     if lisi_graph_:
-        print('LISI graph score...')
-        ilisi_raw, clisi_raw = lisi_graph(
+        clisi_ = True,
+        ilisi_ = True
+
+    if clisi_:
```

Comment:
any reason to not just keep this in `lisi_graph_`? I guess this also allows separate clisi and ilisi runs?

Reply:
Yes, the user then also has the option to run one and not the other. I just added that for consistency with the rest of the behaviour. The old `lisi_graph_` parameter should just work as before